src/Cardano/Wallet/CoinSelection.hs

```diff
@@ -68,3 +78,29 @@ instance Semigroup CoinSelection where
 
 instance Monoid CoinSelection where
     mempty = CoinSelection [] [] []
+
+
+
+-- | Shuffles a list. Meant to be used on @CoinSelection@ outputs.
+shuffle :: [a] -> IO [a]
```

Comment:
Added `shuffle` here

https://github.com/input-output-hk/cardano-wallet/pull/166#discussion_r274322228

Could change the `[a]` to `[TxOut]` or `CoinSelection` I guess

Reply:
No need to be too specific here. `a` is good.